### ISSUE TITLE
System.Text.Json support?

### ISSUE THREAD

**Issue by dpaquette** [ardalis/SmartEnum#71]
Does this package support serialization with `System.Text.Json`?

**Comment by ardalis** [comment]
Not yet - I think a separate package would be needed for it. Should be pretty easy to do at this point given the other examples.

**Comment by PaulRReynolds** [comment]
Hi @ardalis, thanks for this library, it's really helping us out on a few projects. I'd like to serialize SmartEnum with System.Text.Json and I can see this was added in #102, however I don't see a NuGet package for it yet. It looks like there's a NuGet link for this on the home page of this repo, but returns a 404 when clicked. Is this something that's in the pipeline?

 Thanks again, Paul

**Comment by ardalis** [comment]
Yeah, looks like I need to do another NuGet release here soon. I'll try and knock that out this week.

**Comment by murh-lego** [comment]
Is it possible to make it more generic or default at least if we don't have anything fancy? 

 new JsonSerializerOptions ()
 {
     Converters = { 
          new JsonStringEnumConverter(), 
          new SmartEnumNameConverter() <-- this guy is SmartEnumNameConverter<TEnum, TValue>
    }
 }

**Comment by murh-lego** [comment]
> Any update ?

There is support via SmartEnumNameConverter

However, I'd like a bit more approachable API so I can just add it to the Converters.
